### PR TITLE
README: add dendron web client

### PR DESCRIPTION
**Pull request by kevinslin**
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

I made an online version of tldr. It lets folks browse it online as well as import it locally inside VSCode to browse inside [Dendron](https://dendron.so/) - an open source note-taking tool made for devs. There's an `Edit on Github` link at the bottom of each page which will take users to the tldr docs. I'm also working on incorporating translations for a future release

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -142,6 +142,7 @@ both for the command line and for other platforms:
 - [Vim Client](https://github.com/wlemuel/vim-tldr)
 - [Visual Studio Code extension](https://github.com/bmuskalla/vscode-tldr) available on [Visual Studio Code Marketplace](https://marketplace.visualstudio.com/items?itemName=bmuskalla.vscode-tldr)
 - Web clients:
+    - [tldr.dendron.so](http://tldr.dendron.so): https://tldr.dendron.so/
```
Comment (bl-ue) [inline README.md:145]: Can we remove the trailing slash?
```suggestion
    - [tldr.dendron.so](http://tldr.dendron.so): https://tldr.dendron.so
```

**Review by bl-ue — COMMENTED** [review]
Excellent! Do you have a link to the source?

I've left one tiny comment below.

**Comment by kevinslin** [comment]
> Excellent! Do you have a link to the source?
> 
> I've left one tiny comment below.

Yes. https://github.com/kevinslin/seed-tldr. Should I also put that in the README? (the plan is to use github actions to sync the site with the latest from tldr)

**Comment by kevinslin** [comment]
Also, signed the CLA multiple times now but it doesn't seem to be taking :/ 

Fullscreen_3_13_21__5_30_PM

File: README.md [hunk 2]
```diff
@@ -142,6 +142,7 @@ both for the command line and for other platforms:
 - [Vim Client](https://github.com/wlemuel/vim-tldr)
 - [Visual Studio Code extension](https://github.com/bmuskalla/vscode-tldr) available on [Visual Studio Code Marketplace](https://marketplace.visualstudio.com/items?itemName=bmuskalla.vscode-tldr)
 - Web clients:
+    - [tldr.dendron.so](http://tldr.dendron.so): https://tldr.dendron.so
```
Comment (bl-ue) [inline README.md:145]:
```suggestion
    - [tldr.dendron.so](https://github.com/kevinslin/seed-tldr): https://tldr.dendron.so
```

**Review by bl-ue — COMMENTED** [review]
Thank you, how about this, then?

**Comment by bl-ue** [comment]
>Also, signed the CLA multiple times now but it doesn't seem to be taking :/

@kevinslin the CLA isn't working because the email address that you used for the commit (`kevinslin8@gmail.com`) isn't added to your GitHub account.  To fix this, you can either:
1. Change the email you used for the commit† to [one of the emails attached to your GitHub account](https://github.com/settings/emails),

2. Or, preferrably (and much easier), [add the email you used for this commit (`kevinslin8@gmail.com`) to your GitHub account](https://github.com/settings/emails).

†To change the email address of the last commit, run this command:
```
git commit --amend --author="Kevin Lin <your@desired.email>"
```

**Comment by kevinslin** [comment]
okay, force pushed the branch with the correct email and also your suggestions

**Review by bl-ue — APPROVED** [review]
Nice, thank you @kevinslin 👍🏻

**Comment by kevinslin** [comment]
haha, i wondered where all my commits went 😅

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**